src/WebJobs.Script/Config/ScriptSettingsManager.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    public sealed class ScriptSettingsManager
```

Comment:
Since we've decided to move away from having an interface and we have a direct dependency on this type throughout the codebase, this class shouldn't be sealed, as we'll need to derive from it for mocks and alternative implementations.